dts/bindings/ipm/intel,intel-adsp-mailbox.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: INTEL ADSP MAILBOX
+
+compatible: "intel,intel-adsp-mailbox"
```

Comment:
Same complaint as elsewhere -- I'm not super comfortable seeing software protocols end up in devicetree configuration.

Reply:
Already replied above.